.github/workflows/ci.yml

```diff
@@ -14,6 +14,12 @@ env:
   cache-version: v1
 
 jobs:
+  gradle_wrapper_validation:
+    name: gradle-wrapper-validation
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gradle/wrapper-validation-action@v1
```

Comment:
Might be worth adding this line to both jobs.

Having it separate means in case of compromise, the CI is still executing the potentially malicious wrapper code in the other jobs. This might be fine, just wanted to call it out.